livre/core/render/FrameInfo.h

```diff
@@ -31,14 +31,34 @@ namespace livre
 struct FrameInfo
 {
     LIVRECORE_API FrameInfo( const Frustum& frustum,
-                             const uint32_t frameId );
+                             const uint32_t frameId,
+                             const uint32_t id );
 
     Frustum frustum; //!< The current frustum.
-    uint32_t frameId ; //!< The current frame id.
+    uint32_t frameId ; //!< The current frame id for livre data sources.
+    uint32_t id; //!< The unique id for each rendered frame
```

Comment:
I thought I saw it before, but could not find it. Don't amend to keep history, squash all just before merge.

Naming needs to be better then. ;)

frameID -> timeStep ?
id -> frame | renderFrame?